extensions/purchase-label/functions/package.json

```diff
@@ -25,6 +25,7 @@
     "mocha": "^9.1.1",
     "shipengine": "^1.0.0",
     "shipengine-firebase-common-lib": "^1.0.0",
+    "ts-mocha": "^8.0.0",
```

Comment:
Should this, chai & mocha be moved to `devDependencies`? Not a major issue just will get installed when deploying the extension - but not get used 

Same on other extensions below

Reply:
Ah dang, good catch. I used `yarn add` and forgot to add `--dev`. I'll update that.